test/OpenFeature.Benchmarks/OpenFeature.Benchmarks.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Should we run it against net7 as well?

Reply:
I get `BenchmarkDotNet has failed to build the auto-generated boilerplate code` when I try net7.0. Either there's some config I'm missing or BenchmarkDotNet isn't compatible with 7.

I'll merge this and we can always come back to the 7 issue later, if we want.